### PR TITLE
docs(components): add more detail to datepicker usage

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

Got a question of "when should I actually use a datepicker on its own, vs in an InputDate format?"

### Added

- Just a bit more of a tangible example use case for datepicker as a standalone

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/DatePicker/DatePicker.mdx [hunk 1]
```diff
@@ -46,6 +46,9 @@ import { DatePicker } from "@jobber/components/DatePicker";
 Use Datepicker when you are looking to provide the user with a highly visual
 interface for intuitive date selections.
 
+A classic use case for a standalone Datepicker would be as a means to navigate
+a more content-heavy calendar interface.
```
Comment (agglasaur) [inline packages/components/src/DatePicker/DatePicker.mdx:50]: "to navigate **to**"

**Review by agglasaur — CHANGES_REQUESTED** [review]

File: packages/components/src/DatePicker/DatePicker.mdx (hunk 1 above)
Comment (chris-at-jobber) [reply packages/components/src/DatePicker/DatePicker.mdx:50]: I don't think so? I'm thinking of something like this

image

Or this
image

Where the Datepicker is a tool for navigating the calendar, not really navigating _to_ the calendar?

Comment (chris-at-jobber) [reply packages/components/src/DatePicker/DatePicker.mdx:50]: Maybe navigating _within_ the calendar is more clear?

Comment (agglasaur) [reply packages/components/src/DatePicker/DatePicker.mdx:50]: oh i see, yeah i definitely interpreted the original differently

Comment (agglasaur) [reply packages/components/src/DatePicker/DatePicker.mdx:50]: `within` makes sense

Comment (chris-at-jobber) [reply packages/components/src/DatePicker/DatePicker.mdx:50]:
```suggestion
within a larger calendar interface.
```

**Review by agglasaur — APPROVED** [review]

**Pull request merged**